### PR TITLE
Update theme_variables.jinja

### PR DESCRIPTION
**Pull request by nicola-decao**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure to update the docs?   
- [x] Did you write any new necessary tests?  

## What does this PR do?
Fixes links.

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/_templates/theme_variables.jinja [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {%- set external_urls = {
-  'github': 'https://github.com/PytorchLightning/pytorch-torchmetrics',
-  'github_issues': 'https://github.com/PytorchLightning/pytorch-torchmetrics/issues',
+  'github': 'https://github.com/PytorchLightning/metrics',
+  'github_issues': 'https://github.com/PytorchLightning/metrics/issues',
   'contributing': 'https://github.com/PytorchLightning/pytorch-lightning/blob/master/CONTRIBUTING.md',
```
Comment (SkafteNicki) [inline docs/source/_templates/theme_variables.jinja:4]:
```suggestion
  'contributing': 'https://github.com/PyTorchLightning/metrics/blob/master/.github/CONTRIBUTING.md',
```

File: docs/source/_templates/theme_variables.jinja [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
   'governance': 'https://github.com/PytorchLightning/pytorch-lightning/blob/master/governance.md',
```
Comment (SkafteNicki) [inline docs/source/_templates/theme_variables.jinja:5]: link not found
```suggestion

```

File: docs/source/_templates/theme_variables.jinja [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-  'docs': 'https://pytorch-torchmetrics.rtfd.io/en/latest',
+  'docs': 'https://torchmetrics.readthedocs.io/en/latest',
   'twitter': 'https://twitter.com/PyTorchLightnin',
   'discuss': 'https://pytorch-lightning.slack.com',
   'tutorials': 'https://pytorch-lightning.readthedocs.io/en/latest/#tutorials',
```
Comment (SkafteNicki) [inline docs/source/_templates/theme_variables.jinja:9]: we do not really have any tutorials for torchmetrics
```suggestion

```

File: docs/source/_templates/theme_variables.jinja [hunk 4]
```diff
@@ -13,6 +13,6 @@
   'features': 'https://lightning-bolts.rtfd.io/en/latest/',
   'blog': 'https://www.pytorchlightning.ai/blog',
   'resources': 'https://pytorch-lightning.readthedocs.io/en/latest/#community-examples',
```
Comment (SkafteNicki) [inline docs/source/_templates/theme_variables.jinja:15]: no community examples (yet)
```suggestion
```

File: docs/source/_templates/theme_variables.jinja [hunk 5]
```diff
@@ -13,6 +13,6 @@
   'features': 'https://lightning-bolts.rtfd.io/en/latest/',
```
Comment (SkafteNicki) [inline docs/source/_templates/theme_variables.jinja:13]:
```suggestion

```

**Review by Borda — APPROVED** [review]
lgtm, the point here is that if they are not filed, it uses values from the template targeting PL

**Review by SkafteNicki — APPROVED** [review]

**Pull request merged**